server/backend/testspg/router/test_router_idim_proxy.py

```diff
@@ -23,6 +24,7 @@
 
 endPoint_search_idir = f"{apiPrefix}/identity_search/idir"
 endPoint_search_bceid = f"{apiPrefix}/identity_search/bceid"
+endPoint_search_param_application_id = f"&application_id={FOM_DEV_APPLICATION_ID}"
```

Comment:
Didn't check each test case, but I assume `&application_id={FOM_DEV_APPLICATION_ID}"` because it all use FOM_DEV?

Reply:
I use that because the router tests doesn't care about the application, and we pass the application_id just to let the idim uses test url. In the real case, user will select an application first, and then verify the user, the application_id will be passed to the api call as a parameter.